src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.searchForm {
+  margin-top: 20px;
+}
+
+.icon {
+  padding-left: 26px;
+  padding-top: 25px;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+}
+
+.smallIcon {
+  padding-left: 13px;
+  padding-top: 15px;
+  width: 11px;
+  height: 11px;
+}
+
+.searchBar {
+  padding: 0;
+  outline: none;
+  width: 100%;
+  font-family: inherit;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+}
+
+.bigSearchBar {
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.smallSearchBar {
+  padding-left: 33px;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
2 point in checklist, create one base class, and in second one override some properties that you need